addons/binding/org.openhab.binding.yamahareceiver/implementation.md

```diff
@@ -21,5 +21,151 @@ There is a second discovery service, the ZoneDiscoveryService which is created
 and used in createCommunicationObject() Main_Zone thing handler.
 For every other zone a discovery result will be generated and can be added by the user.
 
+## Protocol
```

Comment:
add empty lines after every headline and before and after any code section, see https://www.eclipse.org/smarthome/documentation/development/bindings/docs.html